sdk/storage/Azure.Storage.Files.DataLake/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/f6fab392bec9dbb3ceeb01c4604eba6d929e0074/specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2021-06-08/DataLakeStorage.json
+    - C:\Users\seanmcc\git\azure-rest-api-specs\specification\storage\data-plane\Azure.Storage.Files.DataLake\preview\2021-06-08\DataLakeStorage.json
```

Comment:
TODO fix this when https://github.com/Azure/azure-rest-api-specs/pull/20607 merges

Reply:
Is this changing back now that the PR is merged?

Reply:
fixed.